wiki/Article_Styling_Criteria/en.md

```diff
@@ -316,13 +317,13 @@ For English article titles, if you need to reword the title, you must rename the
 
 #### Sections
 
-All section headings are to be using levels 2 to 5. Section headings are to use sentence case and should never contain styles nor links. In addition to this, section headings must never skip a heading level (e.g. do not go from a level 2 heading to a level 4 heading). The first section heading must be a level 2 heading. Unlike titles, section headings can have small image icons.
+All section headings are to be using levels 2 to 5, never use level 6. Section headings are to use sentence case and should never contain styles nor links. In addition to this, section headings must never skip a heading level (e.g. do not go from a level 2 heading to a level 4 heading). The first section heading must be a level 2 heading. Unlike titles, section headings can have small image icons.
```

Comment:
`are to` -> `should` Multiple instances(4) throughout.

Put `never use level 6` in parenthesis.